fieldservice/models/fsm_order.py

```diff
@@ -110,7 +110,7 @@ def _track_subtype(self, init_values):
     )
     location_directions = fields.Char()
     request_early = fields.Datetime(
-        string="Earliest Request Date", default=datetime.now()
+        string="Earliest Request Date", default=fields.Datetime.now
```

Comment:
are you sure about this ?
It was introduced here https://github.com/OCA/field-service/pull/852

Reply:
IMO the result would be the same, however, I have left the default as it was before.